.github/workflows/publish_draft_releases.yml

```diff
@@ -318,6 +321,11 @@ jobs:
             --no-autorestart \
             -- \
             --address=ws://127.0.0.1:9921
+      -
+        if: ${{ needs.check-for-runtime-upgrade.outputs.do-versions-match == 'false' }}
+        name: fetch previous ${{ matrix.runtime.name }}_runtime.compact.compressed.wasm
+        run: |
+          wget -cO - https://github.com/Manta-Network/Manta/releases/download/v3.1.5-1/${{ matrix.runtime.name }}-runtime-v3151.compact.compressed.wasm > ${{ github.workspace }}/${{ matrix.runtime.name }}_runtime_previous.compact.compressed.wasm
```

Comment:
is hardcoded version number here a good idea?

Reply:
No. but there's no other way. Either hardcoded release to download or hard-coded branch to build it. Unless I'm missing something better. 
We already hardcode the polkadot binary here, so that was another reason for me to brush that concern off.

Reply:
if updating these is part of release QA i'm okay with keeping it that way.
If not, please add it to the docs ( and then send them to me :) )

Reply:
Added a mention to updating these files in the release template